dolphin/service.py

```diff
@@ -63,6 +65,12 @@
                 default=False,
                 help='Wraps the socket in a SSL context if True is set. '
                      'A certificate file and key file must be specified.'),
+    cfg.HostAddressOpt('trap_receiver_addr',
+                       default="0.0.0.0",
```

Comment:
These are already defined as constants then why hard code as "0.0.0.0" and "162" here?

Reply:
Updated